### PR TITLE
Update Good First Issue template

### PR DESCRIPTION
**Pull request by p-wysocki**
### Changes / Reason for changes

Context: https://github.com/openvinotoolkit/openvino/pull/23409

### Related tickets

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/good_first_issue.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ body:
           discussions, guides.
       value: |
           - [Contribution guide - start here!](https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md)
+          - [Intel DevHub Discord channel](https://discord.gg/7pVRxUwdWG) - engage in discussions, ask questions and talk to OpenVINO developers
```
Comment (alexsu52) [inline .github/ISSUE_TEMPLATE/good_first_issue.yml:45]: Who is moderator of Intel DevHub Discord channel? How can I know about discussion relative to NNCF GFI?

Comment (p-wysocki) [reply .github/ISSUE_TEMPLATE/good_first_issue.yml:45]: We're currently using Intel DevHub Discord channel, where me and @mlukasze received moderator status and channels dedicated to GFI. The only way to know if there's a discussion about NNCF is either periodically checking the Discord chat or being notified by someone else that there's a question regarding your component.

Comment (alexsu52) [reply .github/ISSUE_TEMPLATE/good_first_issue.yml:45]: Glad to hear about it! Get notification sounds good. Will you or @mlukasze notify our team about topics/questions relative to NNCF?

Comment (p-wysocki) [reply .github/ISSUE_TEMPLATE/good_first_issue.yml:45]: We'll do our best. Overall Discord is something we're still working on, both technically and organizationally - perhaps in the future we'll be able to provide more convenient ways of notifying developers about discussions relevant to their components. It's not a fully grown "thing" yet.

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**